docs/api/v3.rst

```diff
@@ -0,0 +1,558 @@
+API v3
+======
+
+The Read the Docs API uses :abbr:`REST (Representational State Transfer)`.
+JSON is returned by all API responses including errors
+and HTTP response status codes are to designate success and failure.
+
+
+Authentication and authorization
+--------------------------------
+
+Requests to the Read the Docs public API are for public and private information
+and do require authentication even for requesting public information.
+
+Scopes
+~~~~~~
+
+The API can be accessed by using the HTTP header ``Authorization``
+with a specific ``Token`` that provide access to different resources.
+
+Available scopes
+++++++++++++++++
+
+* ``project``
+   * ``project:admin``: access to change admin settings (similar to Admin tab)
+   * ``project:read``: access to read all the information about the projects
+   * ``project:write``: access to anything related to perform write actions (triggering a Build for this project, activate/deactivate versions, etc)
```

Comment:
How is this different than `admin`?

Reply:
I'm thinking on a scope that allows you to trigger a build, but not to change the name of the project. From my point of view, `admin` will allow you to change anything under the "Admin" tab of your project dashboard.